src/user.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const User = ({ user }) => (
+  <>{user.name}</>
```

Comment:
```suggestion
  <td>{user.name}</td>
```
Можно сделать так, и обойтись без фрагмента